src/coreclr/src/System.Private.CoreLib/src/System/Runtime/CompilerServices/CastHelpers.cs

```diff
@@ -476,5 +479,17 @@ private static CastResult TryGet(nuint source, nuint target)
         slowPath:
             return ChkCastHelper(toTypeHnd, obj);
         }
+
+        [DebuggerHidden]
+        [StackTraceHidden]
+        [DebuggerStepThrough]
+        private static ref byte Unbox(void* toTypeHnd, object obj)
+        {
+            // this will NRE if obj is null, as expected.
```

Comment:
```suggestion
            // this will throw NullReferenceException if obj is null, as expected.
```
The number of people with a decoder ring for NRE is pretty small...